tests/macsec/test_delete_add.py

```diff
@@ -0,0 +1,43 @@
+'''
+
+The test case will verify that forcing a macsec policy delete and add behaves as intended.
+
+Step 1: Attempt to delete in use profile and verify it errors
+Step 2: Attempt to add an already configured profile and verify it errors
+
+'''
+import logging
+
+import pytest
+import time
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.macsec_required,
+    pytest.mark.topology('t2')
+]
+
+
+def test_delete_add_policy(macsec_duthost, startup_macsec, shutdown_macsec, ctrl_links, profile_name, default_priority,
```

Comment:
When this test is called, the assumption is that "profile_name" is already used and referenced in some interface? Can we check that

Reply:
I can add a check, but the test is labeled as macsec required.  Is there a scenario where a macsec test somehow runs on a setup that doesn't have macsec enabled on it?